### PR TITLE
feat: [BD-46] update card docs with content block empty

### PR DESCRIPTION
**Pull request by vlasovmichael**
- Update Card docs with content block empty variants
- JIRA: https://openedx.atlassian.net/browse/PAR-467

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @vlasovmichael! I've created [BLENDED-1104](https://openedx.atlassian.net/browse/BLENDED-1104) to keep track of it in Jira.   More details are on the [BD-46](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3134685276/BD-46+Paragon+Blended+Support+and+Maintenance) project page.


When this pull request is ready, tag your edX technical lead.

File: src/Card/README.md [hunk 1]
```diff
@@ -334,6 +334,62 @@ When using horizontal variant Paragon provides additional component `Card.Body`
...
 
+### Card Content Block Empty
+
+With image
+
+```jsx live
+<Card style={{ width: '25rem' }}>
+  <Card.ImageCap src="https://source.unsplash.com/360x200/?nature,flower" />
+  <Card.Section className="text-center">
+    <h3>Headline</h3>
```
Comment (adamstankiewicz) [inline src/Card/README.md:345]: Per the Figma spec, it looks like the heading is an `h2` in the examples. Should the example here reflect that? Similar question for the "Without image" example.

File: src/Card/README.md [hunk 2]
```diff
@@ -334,6 +334,62 @@ When using horizontal variant Paragon provides additional component `Card.Body`
...
+    <p>This is an optional text description.</p>
+    <Button>Action</Button>
```
Comment (adamstankiewicz) [inline src/Card/README.md:347]: Per the Figma spec, it looks like the button is always using the brand variant. Should these examples' CTA buttons include `variant="brand"`?

**Review by adamstankiewicz — COMMENTED** [review]
Just 2 minor comments, but otherwise LGTM.

File: src/Card/README.md [hunk 3]
```diff
@@ -334,6 +334,62 @@ When using horizontal variant Paragon provides additional component `Card.Body`
 </>
 ```
 
+### Card Content Block Empty
+
+With image
```
Comment (adamstankiewicz) [inline src/Card/README.md:339]: Should these be treated as sub-headings, e.g. `#### With image`?

File: src/Card/README.md (hunk 1 above)
Comment (vlasovmichael) [reply src/Card/README.md]: Fixed to `h2`

File: src/Card/README.md (hunk 2 above)
Comment (vlasovmichael) [reply src/Card/README.md]: You are right. Fixed

File: src/Card/README.md (hunk 3 above)
Comment (vlasovmichael) [reply src/Card/README.md]: Fixed

**Review by adamstankiewicz — APPROVED** [review]

**Comment by edx-webhook** [comment]
@vlasovmichael 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/paragon&entry.867055334=https://github.com/openedx/paragon/pull/1101&entry.1484655318=https://github.com/vlasovmichael&entry.752974735=2022-02-18+13:12&entry.1917517419=2022-03-11+13:20&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**